include/cglm/vec4.h

```diff
@@ -697,14 +698,36 @@ CGLM_INLINE
 float
 glm_vec4_distance(vec4 a, vec4 b) {
 #if defined( __SSE__ ) || defined( __SSE2__ )
-  return glmm_norm(_mm_sub_ps(glmm_load(b), glmm_load(a)));
+  return glmm_norm(_mm_sub_ps(glmm_load(a), glmm_load(b)));
```

Comment:
May I ask what is the benefit[s] of using a first instead of b?

Reply:
There is no real benefit. It is absolutely the same because the subtraction is then raised to the 2nd power, so the result will always be positive. I just like to see a - b because that's the natural order of the function arguments and there is no reason to put b before a. Inverting the natural order suggests that there is a reason why b needs to come before a, which is not the case.

Reply:
Fair enough.